chicken-sloths/bangazon-site-sprint3
# Initializes ctrl-view pairs

# Description
This PR initializes the basic file structure for our `controller-view` pairs. Basic outlining of the necessary functions is also provided. This in no way is exhaustive, but provides a launching off point.

## Related Ticket(s)
N/A

## Problem to Solve
We needed a basic file structure to work off of.
We need a few functions briefly outlined to help get the ball rolling

## Proposed Changes
Initializes:
```
controllers/
      |
      | - - - cartCtrl.js
      | - - - categoryCtrl.js
      | - - - completeOrderCtrl.js
      | - - - managePaymentsCtrl.js
      | - - - mangeProductsCtrl.js
      | - - - productDetailCtrl.js
      | - - - searchCtrl.js
      | - - - settingsCtrl.js

views/
     |
     | - - - partials/
     |             |
     |             | - - - product-card.pug
     |
     | - - - cart.pug
     | - - - category.pug
     | - - - complete-order.pug
     | - - - manage-payments.pug
     | - - - manage-products.pug
     | - - - new-payment-option.pug
     | - - - product-details.pug
     | - - - search.pug
     | - - - settings.pug
```

## Expected Behavior
`NOTE:` This does not resolve how we are going to structure `routes/`. That conversation was tabled yesterday, so it'd be great to revive it soon.

## Steps to Test Solution

1. `npm start`
1. Go to `localhost:8080`
1. Should just get a nice `<h1>Bangazon!</h1>`
1. Go to `localhost:8080/login`, should see login form
1. Go to `localhost:8080/register`, should see register form
